Engine/ResourceManagement/Importer/ModelImporters/AnimationImporter.cpp

```diff
@@ -85,6 +83,7 @@ void AnimationImporter::GetCleanAnimation(const aiNode* root_node, const aiAnima
 		own_format_animation.keyframes.push_back({ static_cast<float>(keyframe.first), keyframe.second });
 	}
 
+	//assert(own_format_animation.keyframes.size() == animation_duration);
```

Comment:
if it is not require anymore better remove it

Reply:
True, changing it right now!